typescript/submissionEvents.d.ts

```diff
@@ -84,6 +84,19 @@ export type PdfSubmissionEventEmailTemplateMapping = {
       text: string
     }
 )
+
+export type BaseCustomPDFFieldMapping = {
+  /** The name of the field that will be replaced in the custom PDF. */
+  replaceableField: string
+}
+
+export type CustomPDFConfiguration = {
+  /** The URL of the PDF that will be used as a template. */
+  pdfUrl: string
```

Comment:
Is the intention to store this file publicly?

Prob better to store it as an `S3Configuration` object.

Reply:
Also need a list of stored PDFs on the form level.

Reply:
Ah, true. This should prob be an id to reference one of those.